### ISSUE TITLE
How to deal with CKEditor (or other WYSIWYG) on text formats?

### ISSUE THREAD

**Issue by indigoxela** [backdrop-contrib/d2b_migrate#25]
Not sure, what the plan is, but currently (in my testing setup) the filtered_html format lost its editor (CKEditor).

I'm aware that the editor setup is quite different between D7 and B, so I'm not sure if transition can be smooth here.
Maybe this has to stay a manual step. Maybe we should give admins a hint after migration completed?

**Comment by docwilmot** [comment]
If you do a more normal upgrade, (wipe a Backdrop database, upload a D7 database in its place and run update.php) does the editor continue working?

**Comment by indigoxela** [comment]
I don't think so, no. The editor setting's always lost. And it might be tricky, anyway, and is only possible based on some "guessing".
A hint after migration completed would be nice, though.

**Comment by irinaz** [comment]
@indigoxela , this is good point.   I will open new issue with list of "hints" that we could add for site builders.  These would include changes to WYSIWYG, layout systems, maybe some others?

**Comment by irinaz** [comment]
@indigoxela , I migrated several sites now, and conversion is very easy because CK5 is in core.   I think that we need to add suggestions in wiki for now and we can close this issue.
